### DISCUSSION TITLE
Influence of mesh size on temperature

### DISCUSSION THREAD

**Discussion by ZQJ269** [firemodels/fds#11468] [General]
Wow, did the Google discussion group move here. 

Well, let me talk about a confusing thing I've come across. 
I've recently been using FDS to calculate a 1m diameter pool fire for heptane, mass_flux is 0.067kg/m2*S as measured by someone else's experiment. 

first I used a 2cm grid in order to debug my input file so that the calculation would be a bit faster. I output the temperature along the central axis of the pool and I found that the temperature trend obtained with the 2cm grid matched the experiment well. 

I then set the grid to 1cm for accuracy, referring to the grid size of the FDS validation file. I found that at this point my temperatures were very different from the experimental ones. It is well known that the Mass_flux of the oil pool is very sensitive to the grid and at a 2cm grid the Mass_flux did not reach my preset value. at a 1cm grid , the Mass_flux reached my preset value. However, the results were not as accurate as with the coarse mesh. I then used the 1m methanol pool fire example and found that the temperature trend was the same for the 1cm grid and the 2cm grid, even though the Mass_flux burned differently when it was stable.

I'm curious as to why there is this discrepancy, is there some other factor that affects it?

**Comment by drjfloyd** [comment]
No, the Google discussion group is still the primary forum. We turned on discussions here to evaluate the feature.

Are you predicting the burning rate (using a MATL reaction) or specifying the burning rate (using HRRPUA or MLRPUA)? Predicting burning rates in a grid independent manner is still an active area of development. Are  you comparing the QUANTITY of TEMPERATURE or THERMOCOUPLE? When comparing against a thermocouple measurement in a test using THERMOCOUPLE is better.

**Reply by ZQJ269** [reply]
Thank you for your reply.

 I am in fact specifying the burn rate. What confuses me is that the calculations for the two grids with a different fuel yield different conclusions. 

I calculated the methanol example using both 1cm and 2cm grids and the temperature profile trends were approximately the same for both grids. Whereas when calculating heptane using the 1cm and 2cm grids, I found that the results for the 2cm grid were much closer to the real (experimental) values, while the trend for the 1cm grid was completely different.
![image](https://user-images.githubusercontent.com/93297504/220905327-08bdf709-3bb3-40b3-b0de-adb903ad77b8.png)

![image](https://user-images.githubusercontent.com/93297504/220905403-76f315e9-10b3-41d9-ab8e-2f9be26437c6.png)

**Comment by drjfloyd** [comment]
I converted this to an issue (https://github.com/firemodels/fds/issues/11470).  Can you attach the two heptane files.  One would expect some differences when the grid changes but assuming the x-axis is the same on the two plots, this is more than I would expect to see.

**Reply by ZQJ269** [reply]
Thank you

I have put my input files for the different grids here.
The 2cm grid is simply a modification of the MESH in the existing input file. By the way, I did the calculations on FDS version 6.3.0.
I did the calculations last night with 6.7.5 and the results are not yet available. I don't know if this is a version issue or not. I am currently working on a pre-FDS version.
[heptane_1m_96_1cm.txt](https://github.com/firemodels/fds/files/10819597/heptane_1m_96_1cm.txt)

**Reply by drjfloyd** [reply]
Let's continue the discussion in the issue linked above.
